src/content/examples/custom-control/MyControl.ts

```
@@ -0,0 +1,58 @@
+		toggleHillshade.addEventListener('click', () => {
+			const newState = this._toggleHillshade();
+			toggleHillshade.textContent = newState ? 'Disable Hillshade' : 'Enable Hillshade';
+		});
+
+		this._container.appendChild(toggleTerrain);
+		this._container.appendChild(toggleHillshade);
+		return this._container!;
+	}
+
+	onRemove() {
+		if (this._container && this._container.parentNode) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)